modules/recommended-requirements-secured-cluster-services.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * installing/acs-recommended-requirements.adoc
+:_content-type: CONCEPT
+[id="recommended-requirements-secured-cluster-services_{context}"]
+= Secured cluster services
+
+Secured cluster services contain the following components:
+
+* Sensor
+* Admission controller
+* Collector
+
+[NOTE]
+====
+Collector component is not included on this page. Required resource requirements are listed on the xref:../installing/acs-default-requirements.adoc#default-requirements-secured-cluster-services-collector_acs-default-requirements[default resource requirements page].
```

Comment:
```suggestion
Information for the Collector component is not included on this page. Required resource requirements are listed on the default resource requirements page.
```

Reply:
We cannot have cross-references in modules, only in assembly files. We can get around this by adding an "Additional resources" section that is tied to the module and that is included in the assembly right after the module is called. The guidelines are here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references

Reply:
I wasn't happy about this reference because it should point to different pages depending on where a module is used. If it's used in cloud_service, then default resources for cloud_service should be linked. If it's in installing, then related default resources in installing tree should be used.

I have changed this a bit. I have removed links from modules and added "Additional resources" in the assembly doc. And in a module, I'm referring to the name used in "Additional resources".